internal/transform/span.go

```diff
@@ -108,6 +124,8 @@ func (m *mapper) Transform(invocationStartedTimestamp int64) telemetry.Span {
 
 		if isStartSpan {
 			lumigoSpan.ID = fmt.Sprintf("%s_started", lumigoSpan.ID)
+			deadline, _ := m.ctx.Deadline()
```

Comment:
What does it mean?

Reply:
this is used to calculate the end value of the start span